### DISCUSSION TITLE
Code Scanning count is wrong in dashboard

### DISCUSSION THREAD

**Discussion by jlundy2** [community/community#61684] [Code Security]
### Select Topic Area

Bug

### Body

The code scanning count in the list in the code security dashboard isn't the same as when the repo is clicked into

![Screenshot 2023-07-24 at 4 20 30 PM](https://github.com/community/community/assets/16428663/362b543d-cf53-4a0e-9bf9-2f881dbfa7c1)
![Screenshot 2023-07-24 at 4 21 07 PM](https://github.com/community/community/assets/16428663/1de429a4-7b85-46bc-8231-80a1125859af)

**Comment by moonfury-ops** [comment]
Thank you
